### PR TITLE
[docstring] Fix docstring for `LlamaConfig`

### PR DESCRIPTION
**Pull request by pavaris-pm**
# What does this PR do?

Fixes #26638  by fixing a typo in docstring of `LlamaConfig`

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@ydshieh @abzdel
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by pavaris-pm** [comment]
@ydshieh can you please review this

**Comment by pavaris-pm** [comment]
@abzdel i already passed all 7 CI test cases after follow your recommendation. Thank you alex!

**Comment by abzdel** [comment]
> @abzdel i already passed all 7 CI test cases after following your recommendation. Thank you alex!

Awesome! Keep me in the loop if anything else comes up.

File: src/transformers/models/llama/configuration_llama.py [hunk 1]
```diff
@@ -58,25 +58,27 @@ class LlamaConfig(PretrainedConfig):
...
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
-        rms_norm_eps (`float`, *optional*, defaults to 1e-12):
+        rms_norm_eps (`float`, *optional*, defaults to 1e-06):
             The epsilon used by the rms normalization layers.
         use_cache (`bool`, *optional*, defaults to `True`):
             Whether or not the model should return the last key/values attentions (not used by all models). Only
-            relevant if `config.is_decoder=True`.
-        tie_word_embeddings(`bool`, *optional*, defaults to `False`):
-            Whether to tie weight embeddings
+            relevant if `config.is_decoder=True`. Whether to tie weight embeddings
```
Comment (ydshieh) [inline src/transformers/models/llama/configuration_llama.py:72]: Should not have `Whether to tie weight embeddings` here. Could you remove it here?
(this is likely a bug in our script).

File: src/transformers/models/llama/configuration_llama.py [hunk 2]
```diff
@@ -58,25 +58,27 @@ class LlamaConfig(PretrainedConfig):
...
+        pad_token_id (`int`, *optional*): Padding token id.
```
Comment (ydshieh) [inline src/transformers/models/llama/configuration_llama.py:73]: Let's put `Padding token id.` in a separate line after `pad_token_id`. Same for several places below.

Comment (pavaris-pm) [reply src/transformers/models/llama/configuration_llama.py:73]: Ok! I will do it. Do I have to rerun fix makeup after these change?

Comment (ydshieh) [reply src/transformers/models/llama/configuration_llama.py:73]: Not necessary. It's better you can check locally first, but anyway CI will tell us if something goes wrong :-)

**Review by ydshieh — APPROVED** [review]
Great, thank you for the contribution @pavaris-pm

**Comment by ydshieh** [comment]
Hi, could you run `make fixup` 🙏 Thank you!

**Comment by pavaris-pm** [comment]
@ydshieh roger that! i will run `make fixup`

**Comment by pavaris-pm** [comment]
@ydshieh already finished running `make fixup`. Now it is passed all CI tests. Can you please review it.

**Review by ydshieh — APPROVED** [review]
Great, thank you again!

**Pull request merged**